gesture/gesture.go

```diff
@@ -131,7 +134,7 @@ func (c *Click) Hovered() bool {
 	return c.entered
 }
 
-// Pressed returns whether a pointer is pressing.
+// Pressed returns whether a pointer is pressing the left mouse button.
```

Comment:
"primary mouse button". Sorry for renaming the buttons while you're working on this change :)